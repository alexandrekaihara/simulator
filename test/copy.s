LOAD TEN
COPY TEN, AUX
SUB TEN
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO: OUTPUT ONE
STOP
TEN: CONST 10
ONE: CONST 1
MINUSONE: CONST -1
AUX: SPACE