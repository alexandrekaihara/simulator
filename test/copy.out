10 15 9 15 18 2 15 8 12 13 17 14 13 16 14 10 1 -1 0